specs/disciplina/witness/api/explorer.yaml

```diff
@@ -172,16 +179,12 @@ definitions:
         format: int64
       prevHash:
         $ref: "#/definitions/Hash"
-  AddressInfo:
+  AccountInfo:
     type: object
     properties:
       balance:
```

Comment:
This is irrelevant, but why don't we return the nonce?

Reply:
we do, I'll update the spec later